iocage/lib/ioc_json.py

```diff
@@ -651,64 +651,50 @@ def json_set_value(self, prop, _import=False, default=False):
 
                 if value == "yes":
                     try:
-                        try:
-                            jail_zfs_dataset = f"{pool}/" \
-                                f"{conf['jail_zfs_dataset']}"
-                            self.zfs_set_property(jail_zfs_dataset, "jailed",
-                                                  "off")
-                        except libzfs.ZFSException as err:
-                            # The dataset doesn't exist, that's OK
-
-                            if err.code == libzfs.Error.NOENT:
-                                pass
-                            else:
-                                # Danger, Will Robinson!
-                                raise
-
-                        self.zfs.get_dataset(old_location).rename(new_location)
-                        conf["type"] = "template"
-
-                        self.location = new_location.lstrip(pool).replace(
-                            "/iocage", iocroot)
+                        jail_zfs_dataset = f"{pool}/" \
+                            f"{conf['jail_zfs_dataset']}"
+                        self.zfs_set_property(jail_zfs_dataset, "jailed",
+                                                "off")
+                    except libzfs.ZFSException as err:
+                        # The dataset doesn't exist, that's OK
 
-                        iocage.lib.ioc_common.logit(
-                            {
-                                "level": "INFO",
-                                "message": f"{uuid} converted to a template."
-                            },
-                            _callback=self.callback,
-                            silent=self.silent)
+                        if err.code == libzfs.Error.NOENT:
+                            pass
+                        else:
+                            # Danger, Will Robinson!
+                            raise
```

Comment:
Since we're handling other exceptions, let's do this here:
```
iocage.lib.ioc_common.logit(
  {
    "level": "EXCEPTION",
    "message": err
  },
  _callback=self.callback)
```
So the user can get a nicely formatted error, and the lib correctly passes the exception.